communications/meetings.md

```diff
@@ -0,0 +1,56 @@
+# Meetings
+
+Use the guides below as a starting point, and adapt for specific situations accordingly.
+
+## Before A Meeting
+
+Objectives:
+- Give all participants as much context as possible before the meeting.
+- _Asynchronously_ resolve as many items as possible before the meeting.
+- Determine, and possibly _reduce_, the scope, and length of the meeting.
+- Determine, and possibly _eliminate_, reasons to participate in the meeting.
+- Prepare the meeting set up to be as convenient as possible for all participants.
+- Crowdsource bookkeeping effort to increase accuracy, and minimize bias.
+
+Guide:
+- Preferably 1 work-day before the meeting, initiate one discussion per agenda item.
+    - The discussion can be a Slack thread, GitHub issue, Asana task, Google Docs, or anything with an interactive "comments" component.
+    - For a recurring meeting with past sessions, use the ones from the previous meeting session as a starting point. Remove ones no longer needed.
+    - For a net-new meeting, the organizer(s) should define the initial scope of items.
+    - Optionally initiate an extra, open-ended discussion to collect additional topics from all participants, graduate solidified ones to have their discussions.
+- In each discussion, participants write down briefing, information, knowledge, and ideas that are relevant.
```

Comment:
Continuation of previous comment, suggestion for this part is have subtitle plus additional part of participants commenting on agenda thread:

```
* Context: In each discussion, participants write down briefing, information, knowledge, and ideas that are relevant as a comment on the agenda thread.
    * For a net-new meeting, organizer should include context to agenda items
    * For a recurring meeting with past sessions, focus on briefings of pending items from the previous meeting session, such as what each has done, has been doing, and plans to do. Give everyone else a sense of where you are and the direction you're heading to. This helps others to determine if there are potential directional conflicts/blocks.
    * Raise relevant questions, problems, obstacles, unknowns, and any other issues. Try to make them demonstratable, by leveraging modern tools such as screen-captured images or videos, and/or steps to reproduce these issues.
    * Preemptively discuss and seek to resolve as many issues and directional conflicts/blocks as possible.

``` 
May recommend providing an example here of how this workflow could look like. I can create a gif to illustrate this :)

Reply:
please do create a gif

Reply:
Gifs found in here: https://drive.google.com/drive/folders/124IuFXJXmx4M6jqCZWnRjnOMkZ4KJR8h?usp=sharing